io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/reader/internal/CSVInstanceCollection.java

```diff
@@ -85,8 +85,21 @@ private void proceedToNext() {
 
 			// initialize reader if necessary
 			if (csvReader == null) {
-				int skipN = reader.getParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES)
-						.as(Integer.class, 0);
+
+				int skipN = 0;
+				Boolean skipType = reader.getParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES)
+						.as(boolean.class);
```

Comment:
I think it would be better to use the wrapper class `Boolean` for the conversion, especially as it is expected here that `null` may be returned.